packages/components-native/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "./dist",
```

Comment:
```suggestion
    "outDir": "./dist/esm",
```
Or update the package.json

Reply:
Good catch! I updated the `package.json`